translations/fr.json

```diff
@@ -1,339 +1,330 @@
+  "building.ability.embark": "Les unités qui se déplacent ici se transforment en bateaux qui peuvent se déplacer sur l'eau.",
+  "building.ability.route": "Il crée des routes commerciales via {0} vers tout autre {1} dans un rayon de 5 cases.",
+  "building.ability.route.and": "et",
+  "building.ability.road": "Construisez des routes pour relier les villes à la capitale. Les villes connectées reçoivent 1 population en récompense. Les routes donnent également un bonus de mouvement pour toutes les unités.",
+  "building.ability.ruin": "Une ruine d'une civilisation ancienne, elle peut contenir des objets précieux ! Allez-y avec une unité pour l'examiner.",
+  "building.ability.patina": "{0} croît avec le temps, essayez donc de les construire le plus tôt possible.",
   "building.ability.limited": "Vous ne pouvez construire qu'un {0} par ville.",
-  "building.ability.unique": "{0} ne peut être construit qu'une fois.",
-  "building.restriction.near": "Doit être construit proche d'un {0}.",
+  "building.ability.unique": "{0} ne peut être construit qu'une seule fois.",
+  "building.restriction.near": "Doit être construit à côté d'un {0}.",
   "building.restriction.on": "Doit être sur une case avec {0}.",
   "building.names.city": "Ville",
   "building.names.ruin": "Ruine",
-  "building.names.monument1": "Hôtel de la Paix",
-  "building.names.monument2": "Tour de la Sagesse",
-  "building.names.monument3": "Grand Marché",
-  "building.names.monument4": "Tombe de l'empereur",
-  "building.names.monument5": "Arche du pouvoir",
-  "building.names.monument6": "Parc de la Fortune",
+  "building.names.monument1": "Autel de la paix",
+  "building.names.monument2": "Tour de sagesse",
+  "building.names.monument3": "Grand Bazar",
+  "building.names.monument4": "Tombeau des empereurs",
+  "building.names.monument5": "Tour de puissance",
+  "building.names.monument6": "Parc de la fortune",
   "building.names.monument7": "Œil de Dieu",
   "building.names.temple": "Temple",
-  "building.names.burnforest": "Brûler la forêt",
+  "building.names.burnforest": "Forêt en feu",
   "building.names.road": "Route",
-  "building.names.customshouse": "Douane portuaire",
-  "building.names.gather": "Récolter les fruits",
+  "building.names.customshouse": "Maison personnalisée",
+  "building.names.gather": "Récolte de fruits",
   "building.names.farm": "Ferme",
-  "building.names.windmill": "Moulin",
+  "building.names.windmill": "Moulin à vent",
   "building.names.fishing": "Pêche",
-  "building.names.whalehunting": "Chasse à la baleine",
-  "building.names.watertemple": "Temple marin",
+  "building.names.whalehunting": "Chasse aux baleines",
+  "building.names.watertemple": "Temple de l'eau",
   "building.names.port": "Port",
   "building.names.hunting": "Chasse",
-  "building.names.clearforest": "Raser la forêt",
-  "building.names.lumberhut": "Maison de bûcheron",
+  "building.names.clearforest": "Forêt claire",
+  "building.names.lumberhut": "Cabane en bois",
```

Comment:
In the game you can clear the forest, not having a light forest... 
For Maison de bucheron it's nearest than the original term in english than cabane en bois...